recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    brew remove --force $(brew list)
+fi
+
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+./configure \
+    --enable-shared \
+    --enable-netcdf-4 \
+    --enable-dap \
+    --without-ssl \
```

Comment:
Is this a correct switch to build libnetcdf? I did not find anything about ssl at unidata (http://www.unidata.ucar.edu/software/netcdf/docs/getting_and_building_netcdf.html) Can you shed some light, @jakirkham